core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -220,6 +222,9 @@ var StoryEditorPage = function() {
   this.expectChaptersListToBe = async function(chapters) {
     await this.expectNumberOfChaptersToBe(chapters.length);
     for (var i = 0; i < chapters.length; i++) {
+      await waitFor.visibilityOf(
+        await chapterTitles.get(i),
```

Comment:
You don't need to `await` here. See the [wiki](https://github.com/oppia/oppia/wiki/End-to-End-Tests#codeowner-checks).